### ISSUE TITLE
Tree and Tree Item - Ensure all active elements receive keyboard focus or can be activated with the keyboard - (2036570421)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5618]
### Module: 
 38 Tree and Tree Item 

### Violation: 
 Ensure all active elements receive keyboard focus or can be activated with the keyboard 

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036570421 

### Severity: 
 8 

### Description: 
 [Issue]
There are actionable elements in the "Children" and "Multi-children" sections that do not receive keyboard focus. Examples include:

- Grandchild 1
- Grandchild 2
- Great-grandchild 1
- Great-grandchild 2
- Great-great-grandchild 1
- Great-great-grandchild 2

[User Impact]
When actionable elements do not receive focus, keyboard-only and screen reader users will not be able to interact with these controls.

[Code Reference] 
 ```
<calcite-tree-item aria-hidden="false" aria-selected="false" role="treeitem" tabindex="0" depth="2" scale="m" calcite-hydrated="">Grandchild 1</calcite-tree-item>
 ``` 

### Note: 
 [Suggestion]
Ensure that all actionable controls are in the focus order. It is recommended to use native HTML controls such as ``` <a href="#"> ```, ``` <button> ```, ``` <input type="text"> ``` etc. as they provide keyboard focus by default.

 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Duplicate to #5619
